src/Base/Rotation.cpp

```diff
@@ -752,6 +768,8 @@ bool Rotation::isSame(const Rotation& q, double tol) const
     // This term can be simplified to
     // 2 - 2*(x1*y1 + ... + x4*y4) so that for the equality we have to check
     // 1 - tol/2 <= x1*y1 + ... + x4*y4
+    // This simplification only work if both quats are normalized
+    // Is it safe to assume that?

```

Comment:
I didn't check again but AFAIR from working in Rotation.cpp yes, the quaternion are always intended to be normalized.

Reply:
Ok. I leave it as is then.